<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddGoods_list</class>
 <widget class="QDialog" name="AddGoods_list">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="add">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>輸入數量與價格後一併上架</string>
   </property>
  </widget>
  <widget class="QLabel" name="shop_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>961</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(61, 61, 61);</string>
   </property>
   <property name="text">
    <string>shop_title</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="how_many">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>761</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>how_many</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="sort_box">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>90</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <item>
    <property name="text">
     <string>all</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>monster</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>magic</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>trap</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DOGE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>YGO</string>
    </property>
   </item>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>129</y>
     <width>920</width>
     <height>561</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>901</width>
     <height>541</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QPushButton" name="search">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>90</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>搜尋</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>70</y>
     <width>190</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="previous_page">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>上一頁</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="next_page">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>90</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>下一頁</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="clear">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>90</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="pageEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>59</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="to_page">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>前往輸入頁數</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
